src/lib/datepicker/datepicker.spec.ts

```diff
@@ -1489,6 +1499,35 @@ describe('MatDatepicker', () => {
     }));
 
   });
+
+  describe('datepicker with custom header', () => {
+    let fixture: ComponentFixture<DatepickerWithCustomHeader>;
+    let testComponent: DatepickerWithCustomHeader;
+
+    beforeEach(fakeAsync(() => {
+      fixture = createComponent(
+        DatepickerWithCustomHeader,
+        [MatNativeDateModule],
+        [],
+        [CustomHeaderForDatepicker]
+      );
+      fixture.detectChanges();
+      testComponent = fixture.componentInstance;
+    }));
+
+    it('should instantiate a datepicker with a custom header', fakeAsync(() => {
+      expect(testComponent).toBeTruthy();
+    }));
+
+    it('should find the standard header selector', fakeAsync(() => {
```

Comment:
nit: 'should find the standard header element'

Reply:
done in https://github.com/angular/material2/pull/10856/commits/ea5e99b27c55c94c706ddebbeaa6fe03ebd17334